crates/cast/tests/e2e/account/deploy.rs

```diff
@@ -141,6 +141,7 @@ async fn test_too_low_max_fee() {
         NETWORK,
         "--accounts-file",
         accounts_file,
+        "--wait",
```

Comment:
just out of curiousity - does that slow down tests in some significant way? I guess ~5 seconds (since sleep if first in the function)?

Reply:
Yup, they were slowed down but thanks to your comment they are not 🔥